requests/README.md

```
@@ -388,6 +388,40 @@ Updates an existing request before approval or rejection with the provided detai
 
 - Body Parameters:
 
+   - **Example of OOO Request:**
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)